### PR TITLE
Did not push image before. Pushing image now.

### PR DESCRIPTION
**Pull request by tfbielawski**
## Description
This is a basic wireframe for the project homepage. 
It includes a header and nav area with placeholder links, a search bar, and a space for a profile image.
It also includes a main content area, and footer.

Pair Programming:
This wireframe was created with Jesse Johns and Vaibhavi Balar
We used Whimsical to create the wireframe

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [x ] My changes generate no new warnings
- [x ] I have checked my code and corrected any misspellings
- [x ] No duplicate code left within changed files
- [x ] Size of pull request kept to a minimum
- [x ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by jessejohns-1 — APPROVED** [review]
Great push, This will make future additions much easier when we start publishing wireframe images.

**Review by VAIBHAVIBALAR — APPROVED** [review]
Good start for wireframes. The homepage wireframe seems to simple and organized. It contains much visual details with logo fixed. The placement and sizing looks good.

**Comment by tfbielawski** [comment]
https://user-images.githubusercontent.com/70349084/139488357-038a07ce-cd80-466c-855c-b2eee46a0943.mp4

**Comment by tfbielawski** [comment]
https://user-images.githubusercontent.com/70349084/139489190-18a34db6-4c80-4bea-915c-f390cd322c34.mp4

**Review by kennymiesner — APPROVED** [review]
Thanks for adding this, Tom. From a workflow perspective, is the idea to push wireframe images to this folder that are in a work in progress, finalized state, or both?

**Review by TrevourGuild — APPROVED** [review]
The wireframe looks great! I really like the design and the simplicity of the homepage! Great Job!

**Comment by tfbielawski** [comment]
> Thanks for adding this, Tom. From a workflow perspective, is the idea to push wireframe images to this folder that are in a work in progress, finalized state, or both?

Im gonna say both because if we have to make a correction or adjustment, we will need to push again.

File: package.json [hunk 1]
```diff
@@ -19,8 +19,8 @@
     "react-scripts": "3.4.1"
   },
   "scripts": {
-    "start": "craco start",
-    "build": "craco build",
+    "start": "craco --max_old_space_size=4096 start",
```
Comment (paulstgermain) [inline package.json:22]: Nice catch patching our Craco issues here!

**Review by paulstgermain — APPROVED** [review]
## Video Walkthrough Review

I love that you walked through one of the files you added to the project visually, and the wireframe you came up with is of _excellent_ quality, but I would have loved more of an explanation regarding the intent for uploading these images to our repo! Usually images like this won't be uploaded to repos unless used directly in the project or decided upon for other reasons, and so I'd love to hear those reasons here!

## Code Review

While there isn't any code to review here, the designs you came up with are simple, but wonderful for that fact and their layout. I would recommend, however, finding a use for these files moving forward if they are to live in our codebase - perhaps in a README documenting design decisions?

Overall though, this is a harmless change, and so I'm happy to approve it!

**Comment by tfbielawski** [comment]
Thanks Paul. I should have explained that by including the wireframes in the repo, it gives everyone a chance to see what our concept looks like, especially the front end people who will be responsible for coding.

**Comment by paulstgermain** [comment]
> Thanks Paul. I should have explained that by including the wireframes in the repo, it gives everyone a chance to see what our concept looks like, especially the front end people who will be responsible for coding.

Not a problem - I appreciate the clarification here, that makes sense to me! Let's create a README for easy viewing of those files, then link to that README in our main README for documentation. 🙂

**Pull request merged**